pages/[...slug].vue

```diff
@@ -23,8 +23,12 @@ onBeforeMount(async () => {
       page.value = response.data.included[0].attributes
       dataLoaded.value = true
     })
-    .catch(() => {
-      navigateTo('/400')
+    .catch((error) => {
+      if (error.response.status === 404) {
+        navigateTo('/400')
```

Comment:
would it make sense to change this to `/404` since it's going there when a 404 is received? This isn't critical, but it might help future developers unfamiliar with the code base.

Reply:
Bono had it already setup for 400, so I was just using what was there, but sure I can rename it!